Zilon.Core/Zilon.Core/WorldGeneration/DirectBitmap.cs

```diff
@@ -1,13 +1,12 @@
 ﻿using System;
 using System.Drawing;
-using System.Drawing.Imaging;
 using System.Runtime.InteropServices;
 
 namespace Zilon.Core.WorldGeneration
 {
     public class DirectBitmap : IDisposable
     {
-        public Bitmap Bitmap { get; private set; }
+        //public Bitmap Bitmap { get; private set; }
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this commented out code.](https://app.codacy.com/manual/kreghek/Zilon_Roguelike/pullRequest?prid=4940399)